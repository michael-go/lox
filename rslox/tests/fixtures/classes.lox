class Brioche {}
print Brioche;
print Brioche();

print "---";

class Toast {}
var toast = Toast();
toast.jam = "grape";
print toast.jam;
print toast.cheese = "gauda";


print "---";

class Pair {}
var pair = Pair();
pair.first = 1;
pair.second = 2;
print pair.first + pair.second;