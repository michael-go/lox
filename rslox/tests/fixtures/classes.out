# exit code: 0
# stdout:
Brioche
Brioche instance
"---"
"grape"
"gauda"
"---"
3

# stderr:

